pgo/workloads/cs-example/run

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+
+trap 'pkill -P $$ || true' EXIT
+
+DIR=`dirname $(realpath "$0")`
+BIN=`realpath "$1"`
+TYPE="$2"
+OUT=`realpath "$3"`
+cd "$DIR"
+
+function wait_for_port {
+    timeout --foreground "$3" bash -c 'until printf "" 2>>/dev/null >>/dev/tcp/'"$1/$2"'; do sleep 0.1 || exit 1; done'
+}
+
+SERVER_ADDRS=(127.42.42.1 127.42.42.2 127.42.42.3)
+SERVER_PIDS=()
+
+for ADDR in "${SERVER_ADDRS[@]}"; do
+    cd "$OUT" # chdir to OUT so that bolt instrumentation leaves its output there
+    env LLVM_PROFILE_FILE="$OUT/$ADDR-%m.profraw" "$BIN" --options-file="$DIR/scylla.yaml" --workdir="$OUT"/workdir-"$ADDR" --ring-delay-ms=1000 --overprovisioned --developer-mode=yes --smp=2 --memory=1G --seed-provider-parameters=seeds="${SERVER_ADDRS[0]}" --listen-address="$ADDR" --rpc-address="$ADDR" --api-address="$ADDR" 2>"$OUT/$ADDR.log" ${BOLT_OPTION:-} &
```

Comment:
--unsafe-bypass-fsync and --kernel-page-cache will reduce dependency on real disk speed. If you rebase then --reactor-backend=io_uring will also help since io_uring works much better with --kernel-page-cache.

Reply:
Noted.